examples/buffer/src/main.rs

```diff
@@ -0,0 +1,33 @@
+use bindings::{Windows::Foundation::*, Windows::Win32::WinRT::IMemoryBufferByteAccess};
+use windows::*;
+
+fn as_slice(buffer: &IMemoryBufferReference) -> Result<&mut [u8]> {
```

Comment:
Wondering where small little helpers like this could potentially live so that users don't have to write them themselves. Not something we need to address right now, but at some point it'd be nice for these little helpers to live somewhere.